src/plugins/globalglob/README.md

```diff
@@ -0,0 +1,13 @@
+- infos = Information about the globalglob plugin is in keys below
+- infos/author = Name <thomas.waser@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/placements = postgetstorage presetstorage
+- infos/description =
+
+## Introduction ##
+
+The Globalglob plugin tries to match every key with keys from the spec namespace using globbing expressions and copies the metadata.
```

Comment:
Is it possible that one `glob` plugin is used for both matching in spec and matching its plugin configuration?